data/json/mutations/mutations.json

```
@@ -1279,9 +1279,9 @@
   {
     "type": "mutation",
     "id": "SCHIZOPHRENIC",
-    "name": { "str": "Schizophrenic" },
+    "name": { "str": "Kaluptic Psychosis" },
```

No, here's how to add one.
```suggestion
    "name": { "str": "Kaluptic Psychosis", "ctxt": "translation help" },
```